### PR TITLE
Fix:code-cleanup

### PR DESCRIPTION
**Pull request by top-gun007**
PR for #134 
Changes:

- removed hard-coded strings.

- removed unused imports.

- used JavaNamingConventions.

- reformatted code

### CODE REVIEW

**Comment by top-gun007** [comment]
Please review @chhavip 
Thank you

**Comment by top-gun007** [comment]
**I also found some memory leaks and some null pointer exceptions should I correct them too.**

File: PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java [hunk 1]
```diff
@@ -14,7 +14,9 @@
 import android.widget.Button;
 import android.widget.ImageButton;
 import android.widget.ImageView;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java]: Remove extra lines everywhere between imports.

File: PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java [hunk 2]
```diff
@@ -11,7 +11,9 @@
 import android.view.View;
 import android.widget.ImageView;
 import android.widget.TextView;
+
 import com.akexorcist.roundcornerprogressbar.IconRoundCornerProgressBar;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/DressingRoomActivity.java]: same

File: PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java [hunk 3]
```diff
@@ -10,6 +10,7 @@
 import android.view.View;
 import android.view.View.OnClickListener;
 import android.widget.Button;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java]: remove

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 4]
```diff
@@ -283,10 +292,12 @@ public void onClick(View v) {
             final ImageView imageViewGlasses = (ImageView) findViewById(R.id.imageViewGlasses);
             final ImageView imageViewHat = (ImageView) findViewById(R.id.imageViewHat);
             final ImageView imageViewNecklace = (ImageView) findViewById(R.id.imageViewNecklace);
+
             final TextView tvHandbagPoints = (TextView) findViewById(R.id.tvHandbagPoints);
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: remove extra line

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 5]
```diff
@@ -283,10 +292,12 @@ public void onClick(View v) {
...
             final TextView tvGlassesPoints = (TextView) findViewById(R.id.tvGlassesPoints);
             final TextView tvHatPoints = (TextView) findViewById(R.id.tvHatPoints);
             final TextView tvNecklacePoints = (TextView) findViewById(R.id.tvNecklacePoints);
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: same. remove everywhere added.

File: PowerUp/app/src/main/res/layout/pgame.xml [hunk 6]
```diff
@@ -56,89 +55,95 @@
...
 
     <TextView
         android:id="@+id/questionView"
-        android:hint="Question"
-        android:layout_marginLeft="20dp"
         android:layout_width="@dimen/question_view_width"
         android:layout_height="wrap_content"
         android:layout_alignParentLeft="true"
         android:layout_alignTop="@id/askerImageView"
+        android:layout_marginLeft="20dp"
```
Comment (chhavip) [inline PowerUp/app/src/main/res/layout/pgame.xml]: move to dimens.xml

**Comment by top-gun007** [comment]
@chhavip I have implemented your suggested changes please merge.

**Comment by chhavip** [comment]
Sqaush the commits please. One feature can have only one commit.

**Comment by top-gun007** [comment]
commits squashed.

**Comment by chhavip** [comment]
@top-gun007 can you please resolve the merge conflicts and submit again? You might require to rebase to make sure there is still only one commit for the feature.

**Comment by top-gun007** [comment]
Seems two extra fields were added in dimen.xml.
Will add those and submit again with the commits squashed.

**Comment by top-gun007** [comment]
@chhavip can you guide me how can I correct this.
Thank you

**Comment by chhavip** [comment]
You will need to rebase your older commits so they come after the work that exists so far in the project and then submit again as one commit. The issue has occurred due to updates in the original project that did not exist when you did your part of the work. You have the option of either fixing this PR or creating a new one with the required changes and no merge conflicts.

**Comment by top-gun007** [comment]
Merge conflict resolved. Please check @chhavip 
Thank you for helping me out with it.

**Pull request merged**